### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rajainal**
Menambahkan detail baru di README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # Belajar-Kolaborasi
 
 Belajar Github sangat menyenangkan.<br>
-Sekarang sedang belajar mendokumentasikan.
+Sekarang sedang belajar mendokumentasikan dengan:
+- squash and merge
+- now code review
```
Comment (MrLemon-Lime) [inline README.md:6]: no need to add "now"

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # Belajar-Kolaborasi
 
 Belajar Github sangat menyenangkan.<br>
-Sekarang sedang belajar mendokumentasikan.
+Sekarang sedang belajar mendokumentasikan dengan:
```
Comment (MrLemon-Lime) [inline README.md:4]: find alternative way to say it. paraphrase.

**Review by MrLemon-Lime — CHANGES_REQUESTED** [review]
fix it first before I can accept the pull request.

**Comment by rajainal** [comment]
already fix the text. pls check

**Review by MrLemon-Lime — APPROVED** [review]
good, aprove

**Comment by MrLemon-Lime** [comment]
good job pal

**Pull request merged**